src/content/effective-dart/documentation.md

```diff
@@ -393,12 +424,22 @@ Flag addFlag(String name, String abbr) => ...
 The convention in Dart is to integrate that into the description of the method
 and highlight parameters using square brackets.
 
+Consider having sections starting with "The \[paramameter\]" to describe 
+parameters, with "Returns" for the returned value and "Throws" for exceptions.
+Errors can be documented the same way as exceptions,
+or just as requirements that must be satified, without documenting the
+precise error which will be thrown.
```

Comment:
Just the guidelines we use in the platform libraries - paragraphs starting with "Returns" or "Throws" are easy to scan for.

Reply:
Let's chat. I'd like to learn more about this rule and the history behind the push to include it. I'm worried that we're overcomplicating things to avoid using 'Returns' in our function summaries. I agree that 'Returns' should not be used but that we probably don't need to go to this extent to define a new rule.